### ISSUE TITLE
Possible Crash When Destroying Orca

### ISSUE THREAD

**Issue by IrateRedKite** [better-modernized-combat/bmod-client#183]
```
Faulting application name: FLServer.exe, version: 0.1.1254.11, time stamp: 0x3ecbb13e
Faulting module name: KERNELBASE.dll, version: 10.0.17763.6054, time stamp: 0x63366e2a
Exception code: 0xe06d7363
Fault offset: 0x00126362
Faulting process id: 0x90c
Faulting application start time: 0x01dae5b5ddb91e32
Faulting application path: C:\bmod-server\Freelancer\EXE\FLServer.exe
Faulting module path: C:\Windows\System32\KERNELBASE.dll
Report Id: bd899ceb-bfa3-4370-a06a-0921999c4c57
Faulting package full name: 
Faulting package-relative application ID: 
```

**Comment by IrateRedKite** [comment]
`E:\FL\Scratch\Source\Server\Space\EqObj.cpp(2519) : *** ERROR: ID(0x1CD) Arch(0xa66a8f8d): Tried to apply damage of type ??? within another damage callback.`
`bm_ju_vhf,2792001421,0xA66A8F8D`

**Comment by IrateRedKite** [comment]
content.dll 0x000bd173 in SP

**Comment by IrateRedKite** [comment]
`bm_npc_aux_acx_1x_xp1_ammo` is defined on `bm_ju_vhf_fc_x_grp_elite`, but is not listed in lootprops.ini, causing this crash. Looking to deploy a fix

**Comment by IrateRedKite** [comment]
Band-aid fix deployed with 3e10fd806acc3eeffa7e3ca556fcd29cc69bc577, we need to have a discussion about the state of + ammo variants, as there are technical hurdles here to be aware of.

**Comment by FJonske** [comment]
Just for the record, since I only just noticed the ping for this in my emails:

Noone really made a mistake here, except me, and noone really needs to change their approach or anything. Lootprops is one of those hybrid things which I meant to automate fully, but never quite did. Consequentially, a bad piece of regex that I committed a while back caused lootprops to not be automatically updated anymore, which was not an issue until now, when that specific piece of ammo was not added to lootprops manually.

The solution here is for me to get off my ass and automate lootprops fully, as there is no reason I can't just generate it from the files in the equipment folder. This should be fairly easy, and should not break anyone's workflows, just make them easier.
